cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -593,7 +595,8 @@ public void getApplicationDetails(FullHttpRequest request, HttpResponder respond
    * Decodes request coming from the {@link #getApplicationDetails(FullHttpRequest, HttpResponder, String)} call.
    */
   private List<ApplicationId> decodeAndValidateBatchApplication(NamespaceId namespaceId,
-                                                                FullHttpRequest request) throws BadRequestException {
+                                                                FullHttpRequest request) throws BadRequestException,
+          ApplicationNotFoundException {
```

Comment:
indentation (make sure your IDE settings are correct)

Reply:
yes, fixing it in the next commit